### PR TITLE
update wiki links in readme to appropriate pages

### PR DESCRIPTION
**Pull request by LakshmiMekala**

### CODE REVIEW

File: cloud/kubernetes/helm/README.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ This chart installs Business Events application deployment on a [Kubernetes](htt
 
 ## Documentation
 
-`How to deploy TIBCO BusinessEvents® application` in kuberenetes using `helm` is available at [wiki pages](https://github.com/TIBCOSoftware/be-tools/wiki).
\ No newline at end of file
+`How to deploy TIBCO BusinessEvents® application` in kuberenetes using `helm` is available at [wiki pages](https://github.com/TIBCOSoftware/be-tools/wiki/Helm%20Chart).
```
Comment (rameshpolishetti) [inline cloud/kubernetes/helm/README.md:9]: @LakshmiMekala making _[wiki pages]_ -> _[Helm Chart]_ would be good

**Review by rameshpolishetti — CHANGES_REQUESTED** [review]

File: cloud/kubernetes/helm/README.md (hunk 1 above)
Comment (LakshmiMekala) [reply cloud/kubernetes/helm/README.md]: text updated

**Review by rameshpolishetti — APPROVED** [review]
LGTM

**Review by vpatil-tibco — APPROVED** [review]
Go ahead

**Pull request merged**